### PR TITLE
feat: update footer.

### PR DESCRIPTION
**Pull request by crhallberg**
image

TODO:
- [x] Add GitHub link to header
- [x] Add OpenHub widget to about page (instead of footer)

### CODE REVIEW

**Comment by demiankatz** [comment]
Thanks, @crhallberg -- we should probably share this with the PMC for feedback (I can pass it along if you don't have all the contact info readily available).

A couple of other thoughts:

1.) We should get approval from all sponsors before moving forward, just in case anyone prefers _not_ to have their logo featured. Perhaps @maccabeelevine could get us an answer for Lehigh; I can look up our Hebis contact once things are a little more settled.

2.) I notice that this PR removes the GitHub banner that actually floats in the header. Is that intentional for accessibility reasons or an accident? Unless there's a specific reason to remove the top banner, I think it might be valid to have links in both places.

**Comment by maccabeelevine** [comment]
Thanks, I'm checking with folks here at Lehigh.

**Comment by maccabeelevine** [comment]
> Thanks, I'm checking with folks here at Lehigh.

Sounds like we are good with it!  Thanks.

**Comment by demiankatz** [comment]
Thanks for the confirmation, @maccabeelevine! I've also reached out to some of our friends at HEBIS (including @ThoWagen) for their input.

**Comment by bsunckel** [comment]
I will check with the VuFind team at HeBIS, but I don't anticipate any objections.

**Comment by bsunckel** [comment]
We (at hebis) are all good with the new footer.

**Comment by crhallberg** [comment]
Updated screenshots:

image

I really wish I had more style control over this widget. (about.html)

image

**Comment by demiankatz** [comment]
@crhallberg, do you think putting something like `<h3>Project Statistics Courtesy of OpenHub</h3>` above the widget would make it look a little less weird?

File: about.html
Comment (demiankatz) [inline about.html]: Slight edit suggestion:

VuFind® is maintained by an international community of developers. Libraries around the world customize and improve VuFind®, contributing those improvements back to project to be shared by all.

Comment (demiankatz) [reply about.html]: (You're missing "an" in front of "international" -- we definitely want to add that. The adjustments to the second sentence are more subjective, but I thought it might make the language a little less redundant).

Comment (crhallberg) [reply about.html]: Much better! I felt redundant from the home page!

**Review by demiankatz — APPROVED** [review]
This looks good to me, but I'll wait until after the PMC meeting to merge in case we get further feedback from anyone there.

**Pull request merged**
